dlt/common/runners/stdout.py

```diff
@@ -24,6 +25,45 @@ def exec_to_stdout(f: AnyFun) -> Iterator[Any]:
             print(encode_obj(rv), flush=True)
 
 
+def iter_std(venv: Venv, command: str, *script_args: Any) -> Iterator[Tuple[int, str]]:
```

Comment:
I know that `shlex` is not windows friendly, I am just wondering if we should sanitize script arguments, wdyt?

Reply:
I'm not sure what I should do here? @sultaniman

Reply:
This was just a security related concern that something might get injected via arguments which will lead RCE or LCE